addons/purchase/tests/test_access_rights.py

```diff
@@ -125,3 +125,17 @@ def test_double_validation(self):
         self.purchase_user.groups_id += group_purchase_manager
         order.with_user(self.purchase_user).button_approve()
         self.assertEqual(order.state, 'purchase')
+
+    def test_create_product_purchase_user(self):
+        self.env['res.config.settings'].create({'group_uom': True}).execute()
+        uom_category_unit = self.env['uom.category'].create({'name': 'test_uom_category_unit'})
+        uom_unit = self.env['uom.uom'].create({'name':'test_unit', 'category_id': uom_category_unit.id})
```

Comment:
`AccountTestInvoicingCommon` inherits from `ProductCommon`, which inherits from `UomCommon`, so instead of creating a new `uom.uom` record for a single test, you could just use one of these as they're not in the default "unit" category:
https://github.com/odoo/odoo/blob/089cf42a5e6c70192a8addf3e3f759ad2b762f6f/addons/uom/tests/common.py#L13-L15